pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -102,21 +102,32 @@ while [ "$#" -gt 0 ]; do
       --use-substitutes|--substitute-on-destination|-s)
         copyFlags+=("-s")
         ;;
-      -I|--max-jobs|-j|--cores|--builders|--log-format)
+      --builders)
         j="$1"; shift 1
         extraBuildFlags+=("$i" "$j")
         ;;
-      --accept-flake-config|-j*|--quiet|--print-build-logs|-L|--no-build-output|-Q| --show-trace|--keep-going|-k|--keep-failed|-K|--fallback|--refresh|--repair|--impure|--offline|--no-net)
+      --I|--max-jobs|-j|--cores|--log-format)
+        j="$1"; shift 1
+        extraBuildFlags+=("$i" "$j")
+        copyFlags+=("$i" "$j")
+        ;;
+      --accept-flake-config|-j*|--quiet|--print-build-logs|-L|--no-build-output|-Q|--show-trace|--refresh|--impure|--offline|--no-net)
+        extraBuildFlags+=("$i")
+        ;;
+      --keep-going|-k|--keep-failed|-K|--fallback|--repair)
```

Comment:
I don't quite understand the criteria, when things are added to copyFlags. i.e. when a machine is already build, what does --keep-going change?

Reply:
Or is this just passing all flags as supported by the command?

Reply:
All supported flags which are the ones mentioned in the manpage and are tested by it not failing

Reply:
@Mic92 is this resolved?

Reply:
Can you also update the manpage accordingly? There is no way to discover these flags otherwise except for reading the code.

Reply:
Something like changing this line: https://github.com/NixOS/nixpkgs/blame/617a1a6e5778df81efceffa656988e38bd293b1c/pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.8#L457

From `Builder options:` to `Builder and nix-copy-closure options:`?

All flags are already mentioned within the manpage, but the language is a bit unclear.

Perhaps one should mention that all flags are passed to the subcommands executed by `nixos-rebuild.sh`?

Reply:
ok.